src/main/scala/firrtl/FileUtils.scala

```diff
@@ -149,6 +149,7 @@ object FileUtils {
   def getLinesResource(resourceName: String): Seq[String] = {
     val inputStream = getClass.getResourceAsStream(resourceName)
     val text = io.Source.fromInputStream(inputStream).getLines().toSeq
```

Comment:
While I realize that this is out scope of this PR, but I think it's in the spirit of this PR... can you switch all of these to use `scala.io.Source`? Annoyingly `io` can be overridden by other libraries/plugins and result in no such method errors if used this way.
```suggestion
     val text = scala.io.Source.fromInputStream(inputStream).getLines().toSeq
```

Reply:
Done!